src/ngraph/op/constant.cpp

```diff
@@ -613,6 +614,27 @@ bool op::Constant::are_all_data_elements_bitwise_identical() const
     return rc;
 }
 
+bool op::v0::Constant::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("element_type", m_element_type);
+    visitor.on_attribute("shape", m_shape);
+    if (m_data == nullptr)
+    {
+        // Filling in a fresh constant
+        allocate_buffer();
+    }
+    DataHandle handle(get_data_ptr_nc(), shape_size(m_shape) * m_element_type.size());
```

Comment:
Who is an owner of this data? As I understand `DataHandle` doesn't control the data.

Reply:
Constant owns the data. DataHandle provides access.

Reply:
Got rid of DataHandle and made things simpler.